src/Command/PopulateIndex.php

```diff
@@ -143,4 +139,47 @@ private function populateIndex(IndexInterface $indexConfig, ElasticaIndex $esInd
         $progressBar->finish();
         $this->output->writeln('');
     }
+
+    private function displayDocumentError(
+        IndexInterface $indexConfig,
+        string $document,
+        AbstractElement $dataObject,
+        \Throwable $throwable,
+    ): void {
+        $this->output->writeln('');
+        $this->output->writeln(sprintf(
+            '<fg=red;options=bold>Error while populating index %s, processing documents of type %s, last processed element ID %s.</>',
+            $indexConfig::class,
+            $document,
+            $dataObject->getId()
+        ));
+        $this->output->writeln('');
+        $this->output->writeln(sprintf('In %s line %d', $throwable->getFile(), $throwable->getLine()));
+        $this->output->writeln('');
+
+        $this->output->writeln($throwable->getMessage());
+        $this->output->writeln('');
+
+        $this->output->writeln($throwable->getTraceAsString());
+        $this->output->writeln('');
+    }
+
+    private function displayIndexError(IndexInterface $indexConfig, \Throwable $throwable): void
+    {
+        $this->output->writeln('');
+        $this->output->writeln(sprintf(
+            '<fg=red;options=bold>Error while populating index %s.</>',
+            $indexConfig::class,
+        ));
+
+        $this->output->writeln('');
+        $this->output->writeln(sprintf('In %s line %d', $throwable->getFile(), $throwable->getLine()));
+        $this->output->writeln('');
+
+        $this->output->writeln($throwable->getMessage());
+        $this->output->writeln('');
+
+        $this->output->writeln($throwable->getTraceAsString());
+        $this->output->writeln('');
+    }
```

Comment:
Duplicate code, perhaps you could optimize that?

Duplicate part
```php
        $this->output->writeln('');
        $this->output->writeln(sprintf('In %s line %d', $throwable->getFile(), $throwable->getLine()));
        $this->output->writeln('');

        $this->output->writeln($throwable->getMessage());
        $this->output->writeln('');

        $this->output->writeln($throwable->getTraceAsString());
        $this->output->writeln('');
```

Reply:
I was considering it and then discarded having a private "display error message" method calling another private "display error message" method, but I guess it makes sense.